apps/user-profile/src/app/components/dialogs/add-auth-img-dialog/add-auth-img-dialog.component.ts

```diff
@@ -30,31 +30,31 @@ export class AddAuthImgDialogComponent implements OnInit {
   ngOnInit(): void {
     this.theme = this.data.theme;
     this.attribute = this.data.attribute;
-    // @ts-ignore
-    this.newImage = this.attribute.value;
+    this.newImage = this.attribute.value as unknown as string;
   }
 
-  handleInputChange(e) {
-    const file = e.dataTransfer ? e.dataTransfer.files[0] : e.target.files[0];
+  handleInputChange(e: Event): void {
+    const element = e.target as HTMLInputElement;
+    const file = element?.files[0];
```

Comment:
I dont think that the changes you have made are just refactor - I think you did here something you didn't want